src/lib.rs

```diff
@@ -258,12 +255,32 @@ pub struct Binding<'a> {
     pub resource: BindingResource<'a>,
 }
 
-/// A description of a bind group layout.
-///
-/// A `BindGroupLayoutDescriptor` can be passed to [`Device::create_bind_group_layout`] to obtain a
-/// [`BindGroupLayout`].
-pub struct BindGroupLayoutDescriptor<'a> {
```

Comment:
I think one reason the descriptor was useful is because of the inheritance chain (`GPUBindGroupLayoutDescriptor : GPUObjectDescriptorBase`). `GPUObjectDescriptorBase` has `DOMString? label` (which we haven't implemented yet), so by accepting a slice here we won't be able to provide debug labels for bind group layouts.

Reply:
aha, great point!